source/options.css

```diff
@@ -46,41 +50,35 @@ li[data-validation] {
 
 details {
 	margin-bottom: 1em;
+	border: solid 1px var(--border);
+	border-radius: 6px;
 }
 
 output {
 	font-style: italic;
 }
 
-details[open] {
-	--border-left: 4px;
-	border-left: solid var(--border-left) #aaa4;
-	padding-bottom: 1px;
-}
-
 summary {
 	--summary-padding: 10px;
-	background: #aaa1;
+	background: var(--background);
 	list-style: none;
 	padding: var(--summary-padding);
-	cursor: pointer;
+	border-radius: 6px;
 }
 
 summary::-webkit-details-marker {
 	display: none; /* Just for Safari. Chrome uses `list-style` */
 }
 
-summary:hover {
-	background: #aaa4;
-}
```

Comment:
No hover style for now, let's see how it feels. I think it's not too difficult to understand that the sections are togglable

Reply:
Maybe add `cursor: pointer` instead?

Reply:
See comment above, I just removed it 😃 

https://marvelapp.com/blog/buttons-shouldnt-hand-cursor/

Reply:
But wasn't the whole point of this PR to match GitHub's style? GitHub does use the pointer cursor for buttons and collapsible sections.

Also, these section elements do not have the "strong accordance" the linked article is referring to, since the default `<details>` styling is changed.